src/components/VmConsoles/tests/SerialConsoleConnector.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
having -2% on code coverage is kind of sad :( Do you think its possible to write some meaningful tests for SerialConsoleConnector ?

Reply:
I've covered VmConsoles, so improved slightly.

The SerialConsoleConnector would require to mock backend by writing much more code than for the implementation. And this code will need to be maintained. I doubt it would bring any real benefit to us.
Let's postpone testing of this till integration tests are ready, so we will be able to connect to a real backend.